swagger.yaml

```diff
@@ -708,6 +708,11 @@ paths:
           description: Ids of categories that should be returned. Multiple categories have to be requested like "?categories[]=1&categories[]=2&categories[]=3"
           required: false
           type: integer
+        - name: private_group
+          in: query
+          description: Uuids of private groups that should be returned. Multiple private groups have to be requested like "?private_groups[]=pg-38e348da-daad-4680-8779-9a450298ca93&private_groups[]=pg-38e348da-daad-4680-8779-9a450298ca93"
```

Comment:
technically a topic can only have 1 friendly_user_test, so adding multiple will result in 0 results when it's AND logic